internal/core/services/operation_manager/operation_manager_test.go

```diff
@@ -617,7 +617,7 @@ func TestListSchedulerActiveOperations(t *testing.T) {
 }
 
 func TestListSchedulerFinishedOperations(t *testing.T) {
-	t.Run("it returns an operation list with finished status", func(t *testing.T) {
+	t.Run("Return an operation list with finished status and its total", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("return an operation list with finished status and its total", func(t *testing.T) {
```
@mflilian We are not using Capital letters for test cases here in Maestro

Reply:
thanks, Gui!